lib/logitech_receiver/base.py

```diff
@@ -267,11 +272,23 @@ def _skip_incoming(handle, ihandle, notifications_hook):
 def make_notification(devnumber, data):
 	"""Guess if this is a notification (and not just a request reply), and
 	return a Notification tuple if it is."""
+
 	sub_id = ord(data[:1])
 	if sub_id & 0x80 == 0x80:
 		# this is either a HID++1.0 register r/w, or an error reply
 		return
 
+	# regular keyboard key press reports and mouse movement reports are not notifications
+	# it would be better to check for report_id 0x20 but that information is not sent here
+	if ( len(data) == _MEDIUM_MESSAGE_SIZE-2  and  ( sub_id==0x02 or sub_id==0x01 ) ) :
+		return
+
+	# connection notifications used to cause unknown notification messages
+	# just ignore them instead
+	if ( len(data) == _MEDIUM_MESSAGE_SIZE-2  and  ( sub_id==0x42 ) ) :
+		if _log.isEnabledFor(_DEBUG):
+			_log.debug("%s: connection message ignored %x %s", devnumber, sub_id, data.hex())
```

Comment:
Do you have to return here?
The spec suggests that this notification allows keys to be released, but that is likely not relevant for Solaar.
https://lekensteyn.nl/files/logitech/Unifying_receiver_DJ_collection_specification_draft.pdf#page=9

Reply:
Yes.  Stupid of me.  A check to not create a notification, but missing the return, so the check is not doing anything.  Fixed.

Reply:
Done